rest-api/src/main/java/uk/gov/hmcts/ccd/definition/store/rest/endpoint/CaseDefinitionController.java

```diff
@@ -142,4 +154,22 @@ public CaseTypeVersionInformation dataCaseTypeVersionGet(@ApiParam(value = "Case
                                                                  String id) {
         return caseTypeService.findVersionInfoByCaseTypeId(id).orElseThrow(() -> new NotFoundException(id));
     }
+
+
+    @GetMapping(value = "/data/jurisdictions/case-type-ids")
```

Comment:
/ data/jurisdictions/case-type-ids --> do we need **jurisdictions** mentioned in url as **case-type-ids** are unique in the system

Reply:
DONE